includes/admin_import.php

```diff
@@ -4,1000 +4,678 @@
+			update_option('tsml_user_interface', $tsml_user_interface);
+			if ($tsml_user_interface == 'tsml_ui') {
+				$tsml_ui = "TSML UI";
+			} else {
+				$tsml_ui = "LEGACY UI";
+			}
+			tsml_alert(__('User Interface Display is now set to <strong>' . $tsml_ui . '</strong>', '12-step-meeting-list') );
+			if ( empty( $tsml_mapbox_key ) && ($tsml_user_interface == 'tsml_ui') ) {
+				tsml_alert(__('<b>Please note</b> that TSML UI only supports Mapbox. To enable mapping you will need a Mapbox token. <br>To sign up for Mapbox <a href="https://www.mapbox.com/signup/" target="_blank">go here</a>. Only a valid email address is required. Copy your access token and paste it in the Mapping & Geocoding section\'s <b>Mapbox Access Token</b> field.', '12-step-meeting-list'),'warning');
+			} 		
+		}
 
-								<div class="input-data-source input-region col-md-2">
-									<?php wp_dropdown_categories(array(
-										'name' => 'tsml_add_data_source_parent_region_id',
-										'taxonomy' => 'tsml_region',
-										'hierarchical' => true,
-										'hide_empty' => false,
-										'orderby' => 'name',
-										'selected' => null,
-										'title' => __('Append regions created by this data source to… (top-level, if none selected)', '12-step-meeting-list'),
-										'show_option_none' => __('Parent Region…', '12-step-meeting-list'),
-									))?>
-								</div>
+		// check user capabilities
+		if ( ! current_user_can( 'manage_options' ) ) {
+			return;
+		}
 
-								<div class="input-data-source input-auto-refresh col-md-2" >
-									<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false)?>
-									<select name="tsml_add_data_source_change_detect" id="tsml_change_detect" >
-									<?php 
-									foreach (array(
-											'disabled' => __('Change Detection Disabled', '12-step-meeting-list'),
-											'enabled' => __('Change Detection Enabled', '12-step-meeting-list'),	
-										) as $key => $value) {?>
-										<option value="<?php echo $key?>"<?php selected($tsml_change_detect, $key)?> ><?php echo $value?></option>
-									<?php }?>
-									</select>
-								</div>
-								<div class="col-md-2">
-									<input type="submit" class="btn button" value="<?php _e('Add Data Source', '12-step-meeting-list')?>">
-								</div>
-							</form>
-						</div>
-					</div>
+		//Get the active tab from the $_GET param
+		$default_tab = null;
+		$tab = isset($_GET['tab']) ? $_GET['tab'] : $default_tab;
+
+		?>
+
+		<style>
+			.column {
+			  vertical-align: top;
+			  padding: 20px 0 0 0;
+			  display: inline-block;
+			  width: 100%;
+			}
+
+			#import_settings_wrap .settings-card {
+				padding: 20px;
+				margin: 10px;
+				line-height: 100%;
+				height: auto;
+			}
+
+			#import_settings_wrap .import-card {
+				margin: 10px;
+				margin-top: 20px;
+				padding: 0 20px 0 20px;
+				line-height: 100%;
+				height: auto;
+			}
+
+			@media only screen and (min-width: 900px) {
+			.column.one-third { width: 33.2%; }
```

Comment:
With the responsive design, the 3rd column drops down below the first 2 columns pretty quickly when I narrow the browser window (on both Firefox and Opera on the Mac). By narrowing the column width, `.column.one-third { width: 33.2%; }`, from 33.2%, to 33%, I can see all three columns much longer, and the imports & settings page looks much better.

Isn't responsive design fun:-)